api/v1alpha1/openstack_ansibleee_types.go

```diff
@@ -139,7 +139,7 @@ type Role struct {
 type Task struct {
 	Name       string     `json:"name"`
 	ImportRole ImportRole `json:"import_role" yaml:"import_role"`
-	Vars       string     `json:"vars,omitempty"`
+	Vars       []string   `json:"vars,omitempty"`
 	When       string     `json:"when,omitempty"`
```

Comment:
should we do the same for `when`?

Reply:
I had no issues with `when`. maybe have to change it later